server/chat/background/summarization.py

```
@@ -311,10 +327,44 @@ def _build_summary_prompt_with_chat(
+                rows = cursor.fetchall()
+                return [
+                    {
+                        "id": row[0],
+                        "title": row[1],
+                        "description": row[2] or "",
+                        "file_path": row[3] or "",
+                        "repository": row[4] or "",
+                    }
+                    for row in rows
+                ]
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)